app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -120,6 +128,8 @@ public static DocumentEditFragment newInstance(File path, boolean pathIsFolder,
     private boolean _firstFileLoad = true;
 
     public DocumentEditFragment() {
+        super();
+        _appSettings = AppSettings.get();
```

Comment:
preferably, please use one of the init / onCreate method overrides to init with fragment context .. avoid AppSettings.get() if possible

Reply:
Fixed